tests/hpc/ww4.pm

```diff
@@ -45,9 +45,10 @@ sub run ($self) {
     record_info "warewulf.conf", script_output("cat /etc/warewulf/warewulf.conf");
     record_info "defaults.conf", script_output("cat /etc/warewulf/defaults.conf");
 
-    $rt = (assert_script_run "wwctl container import docker://registry.suse.de/suse/containers/sle-micro/5.3/containers/suse/sle-micro-rancher/5.3:latest sle-micro-5.3 --setdefault", timeout => 320) ? 1 : 0;
+    my $hpc_container = get_required_var('WAREWULF_CONTAINER');
```

Comment:
Maybe `HPC_WAREWULF_CONTAINER` would be better, but current is still good.

Reply:
Namespacing is good, however, the risk of anything else acquiring the name 'warewulf' is probably low.